gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -178,7 +178,7 @@ public void setUp() throws Exception {
     gobblinMCEWriter = new GobblinMCEWriter(new GobblinMCEWriterBuilder(), state);
   }
 
-  @Test( priority = 3 )
+  @Test( priority = 4 )
```

Comment:
If this test case depends on another test case, we should use "dependsOnMethod", for better readability.